statsmodels/tsa/tests/test_stattools.py

```diff
@@ -170,7 +171,47 @@ def test_acf(self):
     def test_qstat(self):
         #todo why is res1/qstat 1 short
         assert_almost_equal(self.res1[1], self.qstat, DECIMAL_3)
-
+        
+class TestACFMissing(CheckCorrGram):
+    """
+    Test Autocorrelation Function using Missing
+    """
```

Comment:
I don't think is is true for classes, only tests.  There are many examples of `class TestXXX` that have non-empty docstrings, even in this file (in master).  None-the-less I have changed them.